### ISSUE TITLE
Cleanup dependencies

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/ets-wfs20#172]
When running `mvn org.apache.maven.plugins:maven-dependency-plugin:analyze`, I get following warnings:
```
[WARNING] Used undeclared dependencies found:
[WARNING]    org.testng:testng:jar:6.9.10:compile
[WARNING]    org.opengis:geoapi-pending:jar:3.1-M04:compile
[WARNING]    com.sun.jersey:jersey-core:jar:1.19:compile
[WARNING]    org.hamcrest:hamcrest-core:jar:1.3:test
[WARNING]    org.geotoolkit:geotk-utility:jar:3.21:compile
[WARNING]    org.opengis.cite:geomatics-geotk:jar:1.14:compile
[WARNING]    com.beust:jcommander:jar:1.48:compile
[WARNING]    org.geotoolkit:geotk-referencing:jar:3.21:compile
[WARNING]    commons-io:commons-io:jar:1.3.2:compile
[WARNING]    org.geotoolkit.pending:geotk-geometry:jar:3.21:compile
[WARNING]    org.geotoolkit.pending:geotk-temporal:jar:3.21:compile
[WARNING]    org.opengis.cite.saxon:saxon9:jar:9.0.0.8:compile
[WARNING]    com.sun.jersey:jersey-client:jar:1.19:compile
[WARNING]    xml-apis:xml-apis:jar:1.4.01:compile
[WARNING]    xerces:xercesImpl:jar:2.11.0:compile
[WARNING]    javax.ws.rs:jsr311-api:jar:1.1.1:compile
[WARNING] Unused declared dependencies found:
[WARNING]    joda-time:joda-time:jar:2.9.4:compile
[WARNING]    org.apache.commons:commons-io:jar:1.3.2:compile
[WARNING]    com.google.inject:guice:jar:4.0:test
```

All used undeclared compile dependencies shall be declared in Maven pom file.
All unused declared dependencies shall be removed from Maven pom file.

**Comment by dstenger** [comment]
That might be the case. But those dependencies are used by ets-wfs20 code to be able to compile. Thus, it is cleaner to reference them in the project directly.

In addition, we can discuss the fact that one test suite inherits a complete other test suite together with @ghobona and how to tackle resulting problems.

**Comment by keshav-nangare** [comment]
Fixed with #174 PR.
